pkg/kobject/kobject.go

```diff
@@ -107,3 +108,15 @@ type Ports struct {
 	HostIP        string
 	Protocol      api.Protocol
 }
+
+// Volumes holds the volume struct of container
+type Volumes struct {
+	SvcName    string
+	MountPath  string
+	VFrom      string
+	VolumeName string
+	Host       string
+	Container  string
+	Mode       string
+	PVCName    string
+}
```

Comment:
I don't like how many variables there are. It may be better copying how they do it within Docker/CLI so it's easier for us to be compatible with them. See: https://github.com/docker/cli/blob/master/cli/compose/types/types.go#L243 or perhaps even import and use the type.